docs/site/Loopback-component-authorization.md

```diff
@@ -8,18 +8,15 @@ permalink: /doc/en/lb4/Loopback-component-authorization.html
 
 ## Overview
 
-In every web application, we need to have a way to identify access rights of a
-user for any resource, which is known as **Authorization**. This is a
-minimalistic guide for creating such an implementation using Loopback component.
-This can be part of your main REST Application project or can be created as a
-Loopback extension for reuse in multiple projects. Latter is the better option
-for obvious reasons - reusability.
-
-{% include note.html content="
-The LoopBack team is working on making authorization an out-of-the-box
-feature in LoopBack 4. It is a work in progress and will soon be there. Until
-then, this implementation guide can be followed.
-" %}
+Authorization decides if a subject can perform specific action on an object.
+@loopback/authorization is built upon @loopback/authentication to verify whether
+a principal(user, device, or application) has access rights to a resource.
+@loopback/authentication resolves the identity of principal(currently we only
+consider user as the principal) from the request, then passes it to
+@loopback/authorization. @loopback/authorization reads the endpoint’s
+accessibility metadata, like scope and resource name, then execute specified
+authorizers to decide if the identity has access to that particular protected
+resource, returns or rejects accordingly.
```

Comment:
Please quote `@loopback/authorization` and link to https://github.com/strongloop/loopback-next/tree/master/packages/authorization, such as:
```
[@loopback/authorization](https://github.com/strongloop/loopback-next/tree/master/packages/authorization)
```

Reply:
agreed